### ISSUE TITLE
ChronicleMapBuilder.<init> is looking for a target folder

### ISSUE THREAD

**Issue by muthumadi** [OpenHFT/Chronicle-Map#231]
Version:chronicle-map 3.20.2
OS: Mac Catalina 10.15.6
Java: 1.8.0

In our application, we have a custom SecurityManager with a custom policy that restricts read/write access only on a set of a specific folder. 

When I try to create a ChronicleMap, it is scanning the folders trying to look for a target folder and if not found it is trying to create it. While debugging I can see that this logic is in _**net.openhft.chronicle.core.OS.java**_ method _**findTarget()**_

I don't think that the target folder is needed for the operation of the ChronicleMap at runtime.

Here is the exception stack trace (Please note I had to strip out some part of the stack trace)

```
java.lang.ExceptionInInitializerError
	at net.openhft.chronicle.map.ChronicleMapBuilder.<init>(ChronicleMapBuilder.java:210)
	at net.openhft.chronicle.map.ChronicleMapBuilder.of(ChronicleMapBuilder.java:284)
	at net.openhft.chronicle.map.ChronicleMap.of(ChronicleMap.java:72)
...........................
at java.lang.Thread.run(Thread.java:748)
Caused by: java.security.AccessControlException: XXXXXXX does not have permission [read /Users/XXXX/dev/git-repos/release/appserver/target, paths [] ]
```

**Comment by dpisklov** [comment]
You can set 'project.build.directory' to be any folder within the specific folder. Although it's certainly not needed in runtime, it's part of static initializer for a class that is used in many places, and it's easy to override the place being searched.

**Comment by muthumadi** [comment]
I tried setting 'project.build.directory' but it did not help and I think the reason is this

_public static final String TARGET = System.getProperty("project.build.directory", findTarget());_

Though the above line provides the default value for "project.build.directory" ,the findTarget() will get executed no matter if I set the property or not.

On the other side in organizations line ours they take folder access very seriously due to security reasons, and writes are considred serious threat

**Comment by muthumadi** [comment]
Thanks @dpisklov  for the quick turn around

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.20.60](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.20.60), [BOM-2.20.76](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.76)
